config/clusters/2i2c-uk/lis.values.yaml

```diff
@@ -45,10 +45,21 @@ jupyterhub:
       # https://hub.docker.com/r/lisacuk/lishub-base
       name: "lisacuk/lishub-base"
       tag: "581a714"
-    memory:
-      # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
-      guarantee: 512M
-      limit: 2G
+    # Added profileList per https://2i2c.freshdesk.com/a/tickets/1066
+    profileList:
+      - display_name: "Small: ~512 MB RAM / ~0.5 CPU"
+        slug: mem_512m
+        default: true
+        kubespawner_override:
+          # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
+          mem_guarantee: 512M
+          mem_limit: 1G
+      - display_name: "Large: ~1 GB RAM / ~0.5 CPU"
+        slug: mem_1g
+        kubespawner_override:
+          # increase as requested via https://2i2c.freshdesk.com/a/tickets/1066
+          mem_guarantee: 1G
+          mem_limit: 2G
```

Comment:
I'd like to avoid use of relative words like "small" and "large" because it will make communities use these words that we won't understand without reading up on their configuration first.

I think it would make sense for them to use the existing 4 CPU / 32 GB nodes with node resource allocation script generated requests/limits where the limit is 2x the request. Is the script able to do that atm? Hmmm...

Reply:
I actually did a deploy with just the sizes as names and it felt very mathematical 😅 
<img width="801" alt="Screenshot 2023-10-24 at 11 01 57" src="https://github.com/2i2c-org/infrastructure/assets/7579677/2fa49f2a-243e-4bb3-bfd3-d6bd80cc791a">

> I think it would make sense for them to use the existing 4 CPU / 32 GB nodes with node resource allocation script generated requests/limits where the limit is 2x the request. Is the script able to do that atm? Hmmm..

I am not familiar with what the script can do yet 😬